### PR TITLE
New API Version 2017-08-01: Adding Sku property into publicIP and load balancer

### PR DESCRIPTION
**Pull request by gmainar**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by salameer** [comment]
Thank Gloria,

@anuchandy  fire Awwwwaaaayyy

**Comment by anuchandy** [comment]
@ravbhatnagar this is the new API version "2017-08-01". Please refer the commit [Changes for new API version](https://github.com/Azure/azure-rest-api-specs/pull/1528/commits/fc6108bb6780e054f3fd57fe82025c988e2d6108) in this PR to see the diff between this new version and previous version.

**Comment by salameer** [comment]
@anuchandy  can you please provide your feedback from the SDKs as @ravbhatnagar is reviewing to speed us the process here :)

File: specification/network/resource-manager/Microsoft.Network/2017-08-01/loadBalancer.json [hunk 1]
```diff
@@ -1457,6 +1478,10 @@
     },
     "LoadBalancer": {
       "properties": {
+        "sku": {
+          "$ref": "#/definitions/LoadBalancerSku",
+          "description": "The load balancer SKU."
+        },
```
Comment (gmainar) [inline specification/network/resource-manager/Microsoft.Network/2017-08-01/loadBalancer.json]: No

File: specification/network/resource-manager/Microsoft.Network/2017-08-01/publicIpAddress.json [hunk 2]
```diff
@@ -315,6 +332,10 @@
     },
     "PublicIPAddress": {
       "properties": {
+        "sku": {
+          "$ref": "#/definitions/PublicIPAddressSku",
+          "description": "The public IP address SKU."
```
Comment (gmainar) [inline specification/network/resource-manager/Microsoft.Network/2017-08-01/publicIpAddress.json]: No

**Comment by anuchandy** [comment]
The "2017-08-01" api version enables configuring SNAT for back pool VMs and adds SKU for public IP, load balancer. These changes looks good from the SDK side.

File: specification/network/resource-manager/Microsoft.Network/2017-08-01/loadBalancer.json (hunk 1 above)
Comment (anuchandy) [inline specification/network/resource-manager/Microsoft.Network/2017-08-01/loadBalancer.json]: thanks for confirming

**Comment by ravbhatnagar** [comment]
@anuchandy - Looks good. ARM has signed off.

**Review by anuchandy — APPROVED** [review]

**Comment by anuchandy** [comment]
@salameer as discussed, taking this PR to unblock PowerShell and SDKs generations for ignite.. There is a build failure which is related to `cognitiveservices`.

**Pull request merged**